test/steps/RegisterProjectSteps.scala

```diff
@@ -27,23 +27,35 @@ class RegisterProjectSteps extends ScalaDsl with EN with MockitoSugar {
 
 
   Then("""^the projects settings are now$""") { (data: DataTable) =>
-    val expectedProject = data.asList(classOf[Project]).asScala.toList
-    val actualProject = projectRepository.getAll()
-    actualProject mustBe expectedProject
+    val expectedProjects = data.asList(classOf[Project]).asScala
+    val actualProjects = projectRepository.getAll()
+    actualProjects mustBe expectedProjects
+  }
+
+  When("""^a user register a new project with$""") { (data: DataTable) =>
+    cleanDatabase()
```

Comment:
Do not clean the database here, but in a background step at the beginning of the feature